src/colors.config.ts

```diff
@@ -65,6 +65,8 @@ const colors = {
     exportTextareaColor: 'rgba(170, 170, 170, 1)', // #aaa, also used in anchorButton
     inherit: 'inherit',
     currentColor: 'currentColor',
+    alertBorder: 'rgba(36, 36, 36, 1)',
+    alertBg: 'rgba(23, 23, 23, 1)', // #171717
```

Comment:
I'd like to start making design tokens more general, like `bg` and `fg`, so that they can be easily reused. This will also make theming easier. What do you think about `panelBg` and `panelBorder`? This would describe any component that has a background color and border that are slightly off from `bg`.